airbyte-integrations/connectors/destination-duckdb/metadata.yaml

```
@@ -1,8 +1,10 @@
 data:
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.1.0@sha256:bd98f6505c6764b1b5f99d3aedc23dfc9e9af631a62533f60eb32b1d3dbab20c
   connectorSubtype: database
   connectorType: destination
   definitionId: 94bd199c-2ff0-4aa2-b98e-17f0acb72610
-  dockerImageTag: 0.2.1
+  dockerImageTag: 0.3.0
```

```suggestion
  dockerImageTag: 1.0.0
```

It's a breaking change, can we bump be a major version?

---

We spoke offline. This is `community` connector and doesn't receive official support.

Also, DuckDB's own database version files are incompatible across minor versions, and the best way to respect this volatility is to keep pre-1.0 connector status until we have a handle on stability overall.

Last - there's a desire to rewrite this in Java before we give official/certified status to the connector.